README.md

```diff
@@ -4,25 +4,34 @@ Cooper is a tool for Northeastern students to both submit reviews of their co-op
 
 ## Features
 
-- [Next.js](https://nextjs.org) - Web development framework
-- [Tailwind CSS](https://tailwindcss.com) - CSS framework
+- [Next.js](https://nextjs.org/) - Web development framework
+- [tRPC](https://trpc.io/) - End-to-end typesafe API
+- [Tailwind CSS](https://tailwindcss.com/) - CSS framework
 - [shadcn/ui](https://ui.shadcn.com/) - Pre-built components
 - [NextAuth.js](https://next-auth.js.org/) - Authentication
 - [Prisma](https://www.prisma.io/) - ORM
 - [PostgreSQL](https://www.postgresql.org/) - Relational database
-- [Docker](https://www.docker.com/) - Containerization
-- [tRPC](https://trpc.io/) - End-to-end typesafe API
+- [Docker](https://www.docker.com/) - Database containerization for local development
 - [Zod](https://zod.dev/) - Validation
+- [Jest](https://jestjs.io/) - Unit tests
+- [Playwright](https://playwright.dev/) - E2E tests
```

Comment:
i assume playwright/jest will be part of our github actions pipeline once they're built? for the time being, can we indicate that these aren't set up yet in the readme?